_templates/choose-your-wallet.html

```diff
@@ -188,6 +188,69 @@
           privacyaddressreuse: "checkfailprivacyaddressrotation"
           privacydisclosure: "checkfailprivacydisclosurespv"
           privacynetwork: "checkfailprivacynetworknosupporttor"
+
+
+- airbitzwallet:
+    title: "Airbitz Bitcoin Wallet"
+    titleshort: "Airbitz"
+    compat: "mobile android ios"
+    level: 1
+
+    platform:
+      mobile:
+        text: "walletairbitzwallet"
+        link: "https://airbitz.co/bitcoin-wallet/"
+        source: "https://github.com/Airbitz"
+        screenshot: "airbitzwalletandroid.png"
+        os:
+        - android
+        - ios
+        check:
+          control: "checkpasscontrolhybrid"
+          decentralization: "checkfailvalidationcentralized"
```

Comment:
@paullinator This can be replaced with `validation: "checkfailvalidationcentralized"`